### DISCUSSION TITLE
How can we detect a quorum failure using metrics?

### DISCUSSION THREAD

**Discussion by veera-chenna** [rabbitmq/rabbitmq-server#10267] [Questions]
Hi All,

Am looking for the alerting on quorum failures. But when checked at the metrics, couldn't find the correct one. Is there a way to know that quorum failure?

Thanks in advance

**Comment by michaelklishin** [comment]
Queue status is one of queue metrics, and so is the number of replicas it has online.

Note that with a large number of queues you very likely will have to [revert to using aggregates](https://rabbitmq.com/prometheus.html#metric-aggregation) and not per-object (per-queue) metrics. The doc section should explain why.

Before upgrades, there is a [very specific health check](https://www.rabbitmq.com/upgrade.html#quorum-queues) that feels relevant.

**Reply by veera-chenna** [reply]
@michaelklishin , We are using it with less number of queues for now. If needed will retrieve it with metrics family using `/metrics/detailed` endpoint. But cannot find the Queue status metric. Could you please help with the metric name?
